osu.Game.Rulesets.Catch/Edit/Checks/CheckBananaShowerGap.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Game.Rulesets.Catch.Objects;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Edit.Checks.Components;
+using osu.Game.Rulesets.Objects;
+
+namespace osu.Game.Rulesets.Catch.Edit.Checks
+{
+    /// <summary>
+    /// Check the spinner/banana shower gaps specified in the osu!catch difficulty specific ranking criteria.
+    /// </summary>
+    public class CheckBananaShowerGap : ICheck
+    {
+        private static readonly int[] spinner_start_delta_threshold = { 250, 250, 125, 125, 62, 62 };
+        private static readonly int[] spinner_end_delta_threshold = { 250, 250, 250, 125, 125, 125 };
```

Comment:
It's a nitpick but I don't really like the way this is written, kind of hard to tell what I'm looking at. Something like the following would work better, I think:
```csharp
private static readonly Dictionary<DifficultyRating, (int gapBefore, int gapAfter)> = new Dictionary<DifficultyRating, (int, int)>
{
    [DifficultyRating.Easy] = (250, 250),
    [DifficultyRating.Normal] = (250, 250),
    [DifficultyRating.Hard] = (125, 250),
    // ...
}
```

Reply:
Should be better now, thanks for the comment, this is better for sure.